espnet2/tasks/enh.py

```diff
@@ -187,7 +189,74 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             "--use_preprocessor",
             type=str2bool,
             default=False,
-            help="Apply preprocessing to data or not",
+            help="Whether to apply preprocessing to data or not",
+        )
+        group.add_argument(
+            "--speech_volume_normalize",
```

Comment:
Wouldn't be useful to add also an option for varying dynamically the range of level for speech ? 
In dynamic mixing you want often to do that too.

Reply:
Do you mean randomly sampling from a range rather than a fixed value? It sounds reasonable to me.
We can just do that like for `noise_db_range`. @kamo-naoyuki What do you think?

Reply:
Yes that would be good. There are many instances where you want to reduce for example the overall level of the speech utterances e.g. to simulate far field speech (correctly you re scale back after reverberation but sometime you want to simulate the low gain of speech captured by distant mics).

Reply:
I just added support of it (only for EnhPreprocessor).

Reply:
Isn't it better to scale the energy in dB instead of the peak value ? As I see now